driver/driver.go

```diff
@@ -118,7 +132,46 @@ func NewDriver(url, endpoint, nodeID, account string, mounter *mount.SafeFormatA
 		nodeCaps: []csi.NodeServiceCapability_RPC_Type{
 			csi.NodeServiceCapability_RPC_STAGE_UNSTAGE_VOLUME,
 		},
-	}, nil
+		attacher: attacher,
+		quit:     make(chan bool),
+	}
+
+	driver.log.Info("Starting JWT maintainer to refresh the JWT at least once each 30 days.")
+	driver.client.JWT.Get()
+	driver.jwtRefresher = time.NewTicker(29 * 24 * time.Hour)
+	go func() {
+		for {
+			select {
+			case <-driver.quit:
+				return
+			case <-driver.jwtRefresher.C:
+				for {
+					if _, err := driver.client.JWT.Get(); err != nil {
+						driver.log.Errorf("Error refreshing the JWT: %s", err)
+					} else {
+						break
+					}
+					// Sleep 60 seconds before retrying unless quiting
+					for i := 0; i < 60; i++ {
+						select {
+						case <-driver.quit:
+							return
+						default:
+							time.Sleep(1 * time.Second)
+						}
+					}
+				}
+			}
```

Comment:
These nested `select`s could be avoided by using a `time.Timer`, which is configured with a new deadline after expiry - https://golang.org/pkg/time/#Timer.Reset